data-format/message-body.md

```diff
@@ -128,7 +128,7 @@ specified by the [Algorithm Suite ID](#message-header.md#algorithm-suite-id) fie
 
 The last frame in a framed body MUST be a Final Frame.  
 
-The length of the Final Frame MUST be less than or equal to the [Frame Length](#message-header.md#frame-length).
+The length of the plaintext to be encrypted in the Final Frame MUST be less than or equal to the [Frame Length](#message-header.md#frame-length).
```

Comment:
Sorry, given that you were swayed to add the `MUST be greater than or equal to 0` part to the AAD part, I would add it here as well.